.env.development

```
@@ -61,7 +61,7 @@ EMBEDDING_DIM=768
 TOKENIZERS_PARALLELISM=false
 
 # CORS Configuration
-CORS_ORIGINS=http://localhost:3000,http://localhost:8000
+CORS_ORIGINS=http://localhost:3000,http://localhost:8001
 CORS_CREDENTIALS=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CORS_CREDENTIALS key should go before the CORS_ORIGINS key

(UnorderedKey)